ipca/ipca.py

```diff
@@ -71,27 +89,33 @@ def fit(self, Panel=None, PSF=None, refit=False, alpha=0., l1_ratio=1.,
 
             - Column 1: entity id (i)
             - Column 2: time index (t)
-            - Column 3: dependent variable corresponding to observation (i,t)
-            - Column 4 to column 4+L: characteristics.
+            - Column 3 to column 3+L: characteristics.
+
+        y : numpy array
+            dependent variable where indices correspond to those in X
 
         PSF : numpy array, optional
             Set of pre-specified factors as matrix of dimension (M, T)
 
-        refit : boolean, optional
-            Indicates whether the regressor should be re-fit. If set to True
-            the function will skip unpacking the panel into a tensor and
-            instead use the stored values from the previous fit. Note, it is
-            still necessary to pass the previously used P.
+        Gamma : numpy array or None
+            If provided, starting values for Gamma (see Notes)
+
+        Factors : numpy array
+            If provided, starting values for Factors (see Notes)
 
-        alpha : scalar
-            Regularizing constant for Gamma estimation.  If this is set to
-            zero then the estimation defaults to non-regularized.
+        data_type : str
+            label for data-type used for ALS estimation, can be one of:
 
-        l1_ratio : scalar
-            Ratio of l1 and l2 penalties for elastic net Gamma fit.
+            1. panel
+            2. portfolio
```

Comment:
To make the docstring more self-explanatory, can we specify more clearly what we mean by portfolio?